addons/supplies/XEH_postInitClient.sqf

```diff
@@ -3,4 +3,8 @@
 // Exit on Headless
 if (!hasInterface) exitWith {};
 
+[QGVAR(setTraits), {
+    ace_player setUnitTrait [_this, true];
```

Comment:
I would say just pass the player object, as you already have a reference where you call this event.